Gemfile

```diff
@@ -59,6 +59,7 @@ gem "compass-rails",  "2.0.5"
 gem "sass-rails",     "5.0.4"
 gem "autoprefixer-rails", "6.3.6.2"
 gem "bootstrap-switch-rails", "3.3.3"
+gem "font-awesome-rails", "4.4.0.0"
```

Comment:
> Yes, we should.

Meaning drop FA for the publisher or drop Entypo? I see FA is already in Gemfile.lock so I suppose it's used somewhere.